tutor/src/routes.coffee

```diff
@@ -154,6 +157,7 @@ ROUTES = [
   {
     pattern: '/student-preview/:courseId?', name: 'studentPreview',       renderer: getStudentPreview
   }
+  { pattern: 'payments',                name: 'managePayments',           renderer: getPaymentsShell     }
```

Comment:
small thing for later, maybe this should be prefixed with a `/` like the other top level routes?